x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint/experimental_features_disabled.cy.ts

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Should this file name be more policy specific? So we can remove it once the feature flag is finally enabled.

Reply:
Renamed